pkg/controller/deployment/deployment_controller.go

```diff
@@ -555,6 +569,84 @@ func (dc *DeploymentController) getPodMapForDeployment(d *apps.Deployment, rsLis
 	return podMap, nil
 }
 
+func (dc *DeploymentController) enqueueResourceUpdateForRetry() {
+	for key, deploymentToRetry := range dc.deploymentsToReSyncResource {
+		//make sure deployment still alive
+		storedDeployment, err := dc.dLister.Deployments(deploymentToRetry.Namespace).Get(deploymentToRetry.Name)
+		if err != nil || storedDeployment.UID != deploymentToRetry.UID {
+			delete(dc.deploymentsToReSyncResource, key)
+			glog.V(4).Infof("Skipping retry resource update for a non-existing deployment %s", deploymentToRetry.Name)
+			continue
+		}
+		dc.enqueueDeployment(deploymentToRetry)
+		glog.V(4).Infof("Added deployment %s to retry resoruce update", deploymentToRetry.Name)
+	}
+}
+
+func (dc *DeploymentController) patchDeploymentResource(d *apps.Deployment) (bool, error) {
+	selector, err := metav1.LabelSelectorAsSelector(d.Spec.Selector)
+	if err != nil {
+		return false, err
+	}
+	pods, err := dc.podLister.Pods(d.Namespace).List(selector)
+	if err != nil {
+		return false, err
+	}
+
+	cm := controller.NewPodControllerRefManager(dc.podControl, d, nil, controllerKind, nil)
+
+	cMap := make(map[string]*v1.Container)
+	for i, container := range d.Spec.Template.Spec.Containers {
+		cMap[container.Name] = &(d.Spec.Template.Spec.Containers[i])
+	}
+
+	hasResourceUpdate := false
+
+	for _, pod := range pods {
+		// Skip if a resource update is in flight
+		if _, ok := pod.Annotations[schedulerapi.AnnotationResizeResourcesRequest]; ok {
+			glog.Warningf("A resource update is in progress for pod %s. Skipping pod.", pod.Name)
```

Comment:
Please change this to V(2).Infof per my change in the last commit.

Reply:
fixed